src/app/models/sell.model.ts

```diff
@@ -0,0 +1,19 @@
+import { CakeInterface } from './../models/cake.model';
+
+export interface SellInterface {
```

Comment:
As a good practice, you can add the letter `I` before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.

Reply:
Perfect, thanks!